functions/Set-DbaMaxDop.ps1

```diff
@@ -108,10 +108,10 @@ Set recommended Max DOP setting for all databases on server sql2016.
 		}
 		
 		if ($collection -eq $null) {
-			$collection = Test-DbaMaxDop -SqlServer $SqlInstance -SqlCredential $SqlCredential -Verbose:$false
+			$collection = Test-DbaMaxDop -SqlServer $SqlServer -SqlCredential $SqlCredential -Verbose:$false
```

Comment:
I think this might be a missed change, in the current branch Test-DbaMaxDop utilizes SqlInstance as well.